Source/Blazorise/Config.cs

```diff
@@ -51,6 +51,8 @@ public static IServiceCollection AddBlazorise( this IServiceCollection serviceCo
             serviceCollection.AddScoped( mapping.Key, mapping.Value );
         }
 
+        serviceCollection.AddTransient<IRowState, RowState>();
```

Comment:
Still need?

Reply:
Until I replace it with an `EmptyRowState`.